content/links.md

```diff
@@ -227,13 +227,76 @@ instance_groups:
       conn: {as: db_conn}
 ```
 
+### Explicit Linking {: #explicit }
+Explicit linking is defined as specifying your consumer in your deployment manifest. The consumer will be matched on both name and type.
+
+##### Providers
+Explicitly defined providers which can be both explicitly and implicitly consumed, can have the following optional properties:
```

Comment:
> Explicitly defined providers....

The providers are only defined in the job spec of the release (except for custom providers definitions). In the deployment manifest they can only be aliased, exposed to other deployments (with shared `flag`), and blocked (by specifying nil).

Reply:
Maybe changing  `defined` to `specified` ?

Reply:
Will update.